src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -182,13 +190,17 @@ export const useSwapper = () => {
   }, [])
 
   const sellAssetBalance = useAppSelector(state =>
-    selectPortfolioCryptoBalanceByAssetId(state, { assetId: sellTradeAsset?.asset?.assetId ?? '' }),
-  )
-
-  const feeAsset = useAppSelector(state =>
-    selectFeeAssetById(state, sellTradeAsset?.asset?.assetId ?? 'eip155:1/slip44:60'),
+    selectPortfolioCryptoBalanceByAssetId(state, {
+      assetId: sellTradeAsset?.asset?.assetId ?? '',
+    }),
   )
 
+  // TODO: rename to sellFeeAsset
+  const feeAsset = useAppSelector(state => {
+    if (!sellTradeAsset?.asset) {
```

Comment:
i'm also a bit confused about this logic - in what case do we not have a `sellTradeAsset`?

Reply:
It looks like it's on initial load maybe?  This was existing code